xmodule/util/discussion_block.py

```diff
@@ -1,5 +1,5 @@
 """
-Helper functions shared by built-in XBlocks.
+Helper functions for discussion xblock
```

Comment:
Since this file is only used by one other module, I don't think it needs to exist as its own module. Could you just paste its contents into `./xmodule/discussion_block.py`?

Reply:
Apart from `./xmodule/discussion_block.py` it is also being used in `openedx/core/lib/tests/test_xblock_utils.py` that's why I kept it in utils. Should I still go ahead and move the content to `./xmodule/discussion_block.py` and import from there in the other location

Reply:
Yeah that makes more sense. I didn't think about putting them in `openedx/core/lib/xblock_utils` before. I will make the changes shortly.